tests/e2e/controls.py

```diff
@@ -816,9 +816,13 @@ def __init__(
         )
 
     def set(self, value: bool, *, timeout: Timeout = None, **kwargs: object) -> None:
+        self.loc.wait_for(state="visible", timeout=timeout)
```

Comment:
Please remove comment added in #728 about adding these lines throughout controls